src/pages/LoginPage.tsx

```diff
@@ -0,0 +1,127 @@
+  const passwordInputRef = useRef<HTMLInputElement>(null);
+
+  const handleClickShowPassword = () => {
+    if (passwordInputRef.current) {
+      const cursorPosition = passwordInputRef.current.selectionStart;
+      setShowPassword((prev) => !prev);
+      setTimeout(() => {
+        if (passwordInputRef.current) {
+          passwordInputRef.current.setSelectionRange(cursorPosition!, cursorPosition!);
+        }
+      }, 0);
+    }
+  };
+
+  const handleMouseDownPassword = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+  };
+
+  const handleLogin = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    console.log('Email: ', email);
+    console.log('Password: ', password);
+
+    setEmail('');
+    setPassword('');
+  };
+
+  return (
+    <Box
+      sx={{
+        height: '100vh',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        marginLeft: '45vw'
+      }}>
+      <Paper
+        elevation={3}
+        sx={{
+          padding: '3rem',
+          width: '25rem',
+          margin: '2rem 2rem',
+          borderRadius: 3,
+          boxShadow: '0px 4px 15px rgba(0, 0, 0, 0.3)'
+        }}>
+        <Typography
+          variant="h4"
+          gutterBottom
+          align="center"
+          color="primary"
+          sx={{
+            fontSize: '2rem'
+          }}>
+          Login
+        </Typography>
+        <Box
+          component="form"
+          onSubmit={handleLogin}
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            width: '100%',
+            marginBottom: '2rem'
+          }}>
+          <TextField
+            label="Email"
+            type="email"
+            margin="normal"
+            variant="outlined"
+            fullWidth
+            value={email}
+            onChange={(e) => setEmail(e.target.value)}
+            required
+          />
+          <FormControl fullWidth variant="outlined" margin="normal">
+            <InputLabel htmlFor="outlined-adornment-password">Password</InputLabel>
+            <OutlinedInput
+              id="outlined-adornment-password"
+              type={showPassword ? 'text' : 'password'}
+              value={password}
+              inputRef={passwordInputRef}
+              onChange={(e) => setPassword(e.target.value)}
+              endAdornment={
+                <InputAdornment position="end">
+                  <IconButton
+                    aria-label="toggle password visibility"
+                    onClick={handleClickShowPassword}
+                    onMouseDown={handleMouseDownPassword}
+                    edge="end">
+                    {showPassword ? <VisibilityOff /> : <Visibility />}
+                  </IconButton>
+                </InputAdornment>
+              }
+              fullWidth
+              label="Password"
+              required
```

Comment:
Even though both fields are marked as required, I notice only one has the *.

<img width="430" alt="Screenshot 2024-10-07 at 21 29 13" src="https://github.com/user-attachments/assets/259ee262-4022-4b85-8e9e-30af0e61f78b">